_posts/2016-12-02-one-year-on-gcp.md

```diff
@@ -23,40 +23,40 @@ There is no secret that Google knows how to handle large amounts of data. And ma
 
 #### Pubsub
 
-Pubsub is a distributed publish/subscriber queue. It can be used to propagate messages between services, but at Unacast we've mostly used it to regulating back pressure. And it works great in scenarios where you want to buffer traffic/load between front-line API endpoints and sub stream services. We use this approach designing write-centric APIs that can handle large unpredictable spikes of requests. NB! Pubsub doesn’t provide any ordering guarantees, and it doesn’t provide any retention unless a subscription is created for a topic.
+Pubsub is a distributed publish/subscriber queue. It can be used to propagate messages between services, but at Unacast we've mostly used it to regulating back pressure. And it works great in scenarios where you want to buffer traffic/load between front-line API endpoints and sub stream services. We use this approach designing write-centric APIs that can handle large unpredictable spikes of requests. NB! Pubsub doesn’t provide any ordering guarantees, and [it doesn’t provide any retention unless a subscription is created](https://labs.spotify.com/2016/03/03/spotifys-event-delivery-the-road-to-the-cloud-part-ii/) for a topic.
 
 #### BigQuery
 
-BigQuery is a great database for building analytics tools. Storing data is cheap and you only pay for querying and storing data. BigQuery is great because of its out of the box capabilities for querying large amounts of data really fast. To put things into perspective our unscientific tests shows that BigQuery can query 1GB of data as fast as 100GB (and probably even more). One thing to remember when using BigQuery is that it's an append-only database, meaning that you cannot delete single rows only tables<sup>3</sup>. Where Cassandra as row-level TTLs BigQuery ships with table-level TTLs. So implementing data retention has to be done differently and may not be straight forward if you're coming from a standard SQL perspective. 
+BigQuery is a great database for building analytics tools. Storing data is cheap and you only pay for querying and storing data. BigQuery is great because of its out of the box capabilities for querying large amounts of data really fast. To put things into perspective our thorough usage we see that BigQuery can query 1GB of data as fast as 100GB (and probably even more). One thing to remember when using BigQuery is that it's an append-only database, meaning that you cannot delete single rows, only tables<sup>3</sup>. In other words, where Cassandra as row-level TTLs BigQuery ships with table-level TTLs. So implementing data retention has to be done differently and may not be straight forward if you're coming from a standard SQL perspective.
```

Comment:
Change 'To put things into perspective our thorough usage we see that BigQuery can query 1GB of data as fast as 100GB (and probably even more).' to something like:
'To put things into perspective, with our thorough usage we see that BigQuery can query 1GB of data as fast as 100GB (and probably even more).'